src/test/java/uk/gov/digital/ho/hocs/queue/migration/MigrationServiceTest.java

```diff
@@ -67,4 +81,35 @@ public void migrateCase() {
         migrationService.createMigrationCase(migrationData, migrationCaseTypeData);
         verify(migrationCaseworkClient, times(1)).migrateCase(createMigrationCaseRequest);
     }
-}
+
+    @Test
+    public void GivenAValidMigrationMessageWhenCreatingAMigrationRequestThenMigrationResponseContainsAPrimaryCorrespondent() {
```

Comment:
Could you use the same test naming approach as the other test classes? e.g UKVIComplaintServiceTest

Reply:
I've changed them to use the same naming convention as UKVIComplaintServiceTest.